WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -375,7 +376,7 @@ private boolean isViewingNotificationsTab() {
     // Events
 
     @SuppressWarnings("unused")
-    public void onEventMainThread(CoreEvents.UserSignedOut event) {
+    public void onEventMainThread(UserSignedOutWordPressCom event) {
```

Comment:
I think this should be changed to handle `UserSignedOut` separately, like this:
```
@SuppressWarnings("unused")
public void onEventMainThread(UserSignedOutWordPressCom event) {
    resetFragments();
}

@SuppressWarnings("unused")
public void onEventMainThread(CoreEvents.UserSignedOut event) {
    ActivityLauncher.showSignInForResult(this);
}
```
Also, `UserSignedOut` would be more clear if it was renamed `UserSignedOutCompletely`.